src/Sulu/Bundle/AdminBundle/Command/DownloadLanguageCommand.php

```diff
@@ -39,23 +39,66 @@ class DownloadLanguageCommand extends Command
      */
     private $projectDir;
 
-    public function __construct(HttpClientInterface $httpClient, Filesystem $filesystem, string $projectDir)
+    /**
+     * @var string[]
+     */
+    private $defaultLanguages;
+
+    public function __construct(HttpClientInterface $httpClient, Filesystem $filesystem, string $projectDir, array $defaultLanguages)
     {
-        parent::__construct();
         $this->httpClient = $httpClient;
         $this->filesystem = $filesystem;
         $this->projectDir = $projectDir;
+        $this->defaultLanguages = $defaultLanguages;
+        parent::__construct();
     }
 
     protected function configure()
     {
         $this->setDescription('Downloads the currently approved translations for the given language.')
-            ->addArgument('language', InputArgument::REQUIRED, 'The language to download');
+            ->addArgument('languages', InputArgument::IS_ARRAY, 'The language to download', $this->defaultLanguages)
+            ->addOption(
+                'base-url',
+                'b',
+                InputOption::VALUE_REQUIRED,
+                'The endpoint where the translation should be downloaded.',
+                'https://translations.sulu.io/'
+            );
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        if (!class_exists(\ZipArchive::class)) {
+            $ui = new SymfonyStyle($input, $output);
+            $ui->error('The "zip" php extension is required for this command.');
+
+            return 1;
+        }
+
+        $languages = $input->getArgument('languages');
+
+        foreach ($languages as $language) {
+            $this->executeLanguage($input, $output, $language);
+        }
+
+        return 0;
     }
 
-    protected function execute(InputInterface $input, OutputInterface $output)
+    protected function executeLanguage(InputInterface $input, OutputInterface $output, string $language): void
     {
-        $language = $input->getArgument('language');
+        $translationBaseUrl = $input->getOption('base-url');
+
+        $ui = new SymfonyStyle($input, $output);
+        $ui->section('Language: ' . $language);
+
+        if (in_array($language, ['en', 'de'])) {
+            $ui->note(sprintf(
+                'The language "%s" is a core language and doesn\'t need to be downloaded.',
```

Comment:
`Download skipped because Sulu includes a translation for the language "%s" per default.`